docs/static/include/javapluginpkg.asciidoc

```diff
@@ -19,6 +19,11 @@ The Java plugin should be compiled and assembled into a fat jar with the
 into a single jar and write it to the correct folder for later packaging into a
 Ruby gem.
 
+To build the jar file, run the following command from the plugin directory:
+-----
+./gradlew build vendor
+-----
+
```

Comment:
@renshuki, the `vendor` task was mentioned in the preceding paragraph though I suppose it does not hurt to be explicit about it and show the entire command. That said, technically it should be `./gradlew vendor` without the `build` task since `vendor` defines `build` as one of its dependencies.

Reply:
Thanks for your update on this and the correction regarding the command.
It personally took me a while to find how to **Compile to JAR file** and some of my peers struggled as well. I would appreciate if this could be merged.
I've made the change and squashed the unnecessary commit.